### PR TITLE
Added aws-ec2-cw-metrics.md

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

Added new EC2-CW metrics.md

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/observability/aws/integrations/aws-ec2-cloudwatch-metrics.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+id: aws-ec2-cloudwatch-metrics
+title: AWS EC2 CloudWatch Metrics
+sidebar_label: AWS EC2 CloudWatch Metrics
+description: The Sumo Logic App for AWS EC2 (CloudWatch Metrics) allows you to display and gain insight about your EC2 instance metrics (CloudWatch) for cpu, disk, network, EBS and Health Status Check.
+---
+
+[Amazon Elastic Compute Cloud (Amazon EC2)](https://aws.amazon.com/ec2/) provides scalable computing capacity in the Amazon Web Services (AWS) Cloud. You can use Amazon EC2 to launch as many or as few virtual servers as you need, configure security and networking, and manage storage.
+
+The Sumo Logic App for AWS EC2 (CloudWatch Metrics) allows you to collect your EC2 instance metrics and display them using predefined dashboards. The App provides dashboards to display analysis of EC2 instance metrics for CPU, disk, network, EBS, and Health Status Check. Furthermore, it provides detailed insights into all CloudTrail audit events associated with EC2 instances and specifically helps identify changes, errors, and user activities.
+
```
Comment (kimsauce) [inline docs/observability/aws/integrations/aws-ec2-cloudwatch-metrics.md:11]: I'd add a note here and [here](https://help.sumologic.com/docs/observability/aws/integrations/aws-ec2-metrics/) differentiating EC2 from EC2 Cloudwatch.

**Review by kimsauce — DISMISSED** [review]
suggestion in comment

File: docs/observability/aws/integrations/aws-ec2-cloudwatch-metrics.md (hunk 1 above)
Comment (JainM6) [reply docs/observability/aws/integrations/aws-ec2-cloudwatch-metrics.md:11]: The AWS EC2 metrics page will be removed, it is no longer required. I have just added a new file "aws-ec2-cloudwatch-metrics.md". Will remove the aws-ec2-metrics.md. 
Adding more information: Note to customers is not required in this doc as they are aware EC2 Metrics will now be called CW Metrics. Earlier, In aws-ec2-metrics. md, host metrics content was copy pasted, but that's not correct. CW and Host metrics are the source for EC2 metrics, so we just need two pages: aws-ec2-cloudwatch-metrics.md and aws-host-metrics.md.

**Review by stacykor — COMMENTED** [review]
Please incorporate comments from Alekh.

File: docs/observability/aws/integrations/aws-ec2-cloudwatch-metrics.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+| count as count by error_code | sort by count, error_code asc | limit 10
+```
+
+## Viewing AWS EC2 CloudWatch Metrics Dashboards
+
+This page provides examples and descriptions for each of the AWS Observability EC2 CloudWatch Metrics pre-configured dashboards.
+
+[Amazon Elastic Compute Cloud (Amazon EC2)](https://aws.amazon.com/ec2/) is a web service that provides secure, resizable compute capacity in the cloud, enabling you with complete control of your computing resources.
+
+The Sumo Logic App for AWS EC2 allows you to collect your EC2 instance metrics along with  and display them using predefined dashboards. The App provides dashboards to display analysis of EC2 instance metrics for cpu, disk, network, EBS and Health Status Check. Also it provides detailed insights into all CloudTrail audit events associated with EC2 instances and specifically helps identify changes, errors, and user activities
```
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/observability/aws/integrations/aws-ec2-cloudwatch-metrics.md:102]: @JainM6 ☝️

File: docs/observability/aws/integrations/aws-ec2-cloudwatch-metrics.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+
+Use this dashboard to:
+
+* Monitor the geo location for successful and failed events.
+* Observed the event status and top error codes.
+* Track distribution and top event types and trends.
+* Monitor top IAM Users, Assumed Role Users and User agents.
+* Monitor distribution of Successful and failed events with the list of latest events.
+
+![2.1 AWS EC2 Events (CloudTrail).png](https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/AWS-EC2-CW-Metrics/2.1-AWS-EC2-Events.png)
```
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/observability/aws/integrations/aws-ec2-cloudwatch-metrics.md:160]: @JainM6 ☝️

[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/observability/aws/integrations/aws-ec2-cloudwatch-metrics.md:160]: Hi @JainM6 , following up - can you please address this comment?

[the remark this answers is not in the document]
Comment (JainM6) [reply docs/observability/aws/integrations/aws-ec2-cloudwatch-metrics.md:160]: > @JainM6 , please apply [sumoanema](https://github.com/sumoanema)'s feedback

This PR can be closed as all the updates are there on PR  #956.

**Comment by JainM6** [comment]
This PR can be closed as all the updates are there on PR  #956.

**Pull request closed without merging**